web/src/compose_state.ts

```diff
@@ -25,6 +26,14 @@ export function is_recipient_edited_manually(): boolean {
     return recipient_edited_manually;
 }
 
+export function set_restored_message_is_unedited(flag: boolean): void {
+    restored_message_is_unedited = flag;
+}
+
+export function is_restored_message_is_unedited(): boolean {
+    return restored_message_is_unedited;
+}
```

Comment:
I think maybe `set_is_content_unedited_restored_draft` / `get_is_content_unedited_restored_draft` would be more readable names for this?

Reply:
Yeah I was having trouble thinking of good names, so I'm down to switch to that.

Reply:
Actually I like `is_restored_draft_content_unedited` better -- thoughts on that?

Reply:
I think that has the implicature that the compose box has "restored draft content" already, and we're checking if it is unedited. "is_content_unedited_restored_draft" does not have that problem.

Reply:
Sure, I'll do `is_content_unedited_restored_draft` then